### PR TITLE
Ad7134

### PR DESCRIPTION
**Pull request by cristiantpop**
Add makefile support and some small fixes

### CODE REVIEW

File: projects/ad713x_fmcz/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+TARGET := ad713x_fmcz
+ifeq ($(OS), Windows_NT)
+include ../../tools/scripts/windows.mk
+else
+include ../../tools/scripts/linux.mk
+endif
```
Comment (mchindri) [inline projects/ad713x_fmcz/Makefile:6]: nitpick: Looks better on git if extra line is added

File: projects/ad713x_fmcz/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+TARGET := ad713x_fmcz
```
Comment (mchindri) [inline projects/ad713x_fmcz/Makefile:1]: Curiosity: Shouldn't TINYIIOD be added here?

Comment (cristiantpop) [reply projects/ad713x_fmcz/Makefile:1]: No, this Makefile builds `ad713x_fmcz` project. For building project with iio you can run `make TINYIIOD=y`

**Comment by cristiantpop** [comment]
V2:

- Fix typo in commit message: 'spi enfine'
- Add extra line after: `#include "spi.h"`
- Fix commit message: 'will come' is not needed
- Add extra line at the end of Makefile

**Pull request merged**